narwhals/expression.py

```diff
@@ -1871,7 +1871,6 @@ def millisecond(self) -> Expr:
             >>> df_pl = pl.DataFrame(data)
 
             We define a dataframe-agnostic function:
-
```

Comment:
i think the empty line is still needed in order for the example to render correctly?